packages/safe-apps-sdk/src/safe/safe.test.ts

```diff
@@ -25,6 +26,200 @@ describe('Safe Apps SDK safe methods', () => {
     });
   });
 
+  describe('SDK.safe.calculateSafeMessageHash', () => {
+    test('Should generate a valid message hash', async () => {
+      const safeInfoSpy = jest.spyOn(sdkInstance.safe, 'getInfo');
+      safeInfoSpy.mockImplementationOnce(
+        (): Promise<SafeInfo> =>
+          Promise.resolve({
+            chainId: 4,
+            safeAddress: '0x9C6FEA0B2eAc5b6D8bBB6C30401D42aA95398190',
+            owners: [],
+            threshold: 1,
+          }),
+      );
+      // to test message/hash I signed a test message on rinkeby
+      // https://dashboard.tenderly.co/tx/rinkeby/0x9308fb61d9f4282080334e3f35b357fc689e06808b8ad2817536813948e3720d
+      const message = '0x617070726f76652072756770756c6c0000000000000000000000000000000000'; // ethers.utils.formatBytes32String('approve rugpull')
+      const expectedHash = '0x2abe5973348b08db234e452b84bd3fbc74ee544962ff70903e85725d8f5f8eaf';
+      const hash = await sdkInstance.safe.calculateSafeMessageHash(message);
+
+      expect(hash).toEqual(expectedHash);
+    });
+  });
+
+  describe('SDK.safe.check1271Signature', () => {
+    test('Should return true if the message is signed and magic value is returned', async () => {
+      const safeInfoSpy = jest.spyOn(sdkInstance.safe, 'getInfo');
+      // @ts-expect-error method is private but we are testing it
+      const rpcCallSpy = jest.spyOn(sdkInstance.safe.communicator, 'send');
+      safeInfoSpy.mockImplementationOnce(
+        (): Promise<SafeInfo> =>
+          Promise.resolve({
+            chainId: 4,
+            safeAddress: '0x9C6FEA0B2eAc5b6D8bBB6C30401D42aA95398190',
+            owners: [],
+            threshold: 1,
+          }),
+      );
+      rpcCallSpy.mockImplementationOnce(() =>
```

Comment:
Should we check that it is called with the correct data?

Reply:
Done now, could you please check?